.bowerrc

```diff
@@ -1,3 +1,3 @@
 {
-    "directory": "hu_web/hu_browser/static/hu_browser/bower/"
+    "directory": "hu_web/common-static/bower/"
 }
```

Comment:
Okay, after thinking about this for a minute while I was getting the Frosted Flakes out of the pantry, I think you're right that the situation's is different because bower is managing that directory. I like it